pubsub/redis/metadata.go

```diff
@@ -5,9 +5,17 @@
 
 package redis
 
+import (
+	"time"
+)
+
 type metadata struct {
-	host       string
-	password   string
-	consumerID string
-	enableTLS  bool
+	host              string
+	password          string
+	consumerID        string
+	enableTLS         bool
+	processingTimeout time.Duration
+	reclaimInterval   time.Duration
```

Comment:
Please, create/update an issue mentioning this new fields, so they can be in the release notes.

Reply:
Will add to https://github.com/dapr/components-contrib/issues/710